tests/index.ts

```diff
@@ -0,0 +1,33 @@
+import { Application } from '../application.ts'
+import { assertEquals } from "https://deno.land/std/testing/asserts.ts";
```

Comment:
for stability, can you use tagged releases URL, please?
```suggestion
import { assertEquals } from "https://deno.land/std@0.57.0/testing/asserts.ts";
```